src/ResourceManager/Network/Commands.Network/VirtualNetworkGatewayConnection/UpdateAzureVirtualNetworkGatewayConnectionCommand.cs

```diff
@@ -31,6 +32,23 @@ public class SetAzureVirtualNetworkGatewayConnectionCommand : VirtualNetworkGate
             ValueFromPipeline = true,
             HelpMessage = "The VirtualNetworkGatewayConnection")]
         public PSVirtualNetworkGatewayConnection VirtualNetworkGatewayConnection { get; set; }
+        
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Whether to establish a BGP session over a S2S VPN tunnel")]
+        public string EnableBgp { get; set; }
```

Comment:
@cormacpayne This is not the desired behavior. Lack of a switch parameter will mean that BGP will be disabled, which will break existing use cases.